classes/genre.rb

```diff
@@ -0,0 +1,9 @@
+class Genre
+  attr_accessor :name
+
+  def initialize(name, id = SecureRandom.uuid)
```

Comment:
I think the "id" shouldn't be an optional parameter. It always has to be generated by SecureRandom to avoid repeating the id.

Reply:
That is totally true! So, you can merge!